packages/cli/src/commands/doctor/healthchecks/python.ts

```diff
@@ -0,0 +1,59 @@
+import {fetchToTemp} from '@react-native-community/cli-tools';
+import versionRanges from '../versionRanges';
+import {doesSoftwareNeedToBeFixed} from '../checkInstallation';
+import {logManualInstallation} from './common';
+import {HealthCheckInterface} from '../types';
+
+import {updateEnvironment} from '../../../tools/environmentVariables';
+import {join} from 'path';
+import {Ora} from 'ora';
+import {executeCommand} from '../../../tools/executeWinCommand';
+
+export default {
+  label: 'Python',
+  getDiagnostics: async ({Languages}) => ({
+    needsToBeFixed: doesSoftwareNeedToBeFixed({
+      version: Languages.Python.version,
+      versionRange: versionRanges.PYTHON,
+    }),
+
+    version: Languages.Python.version,
+    versionRange: versionRanges.PYTHON,
+  }),
+  win32AutomaticFix: async ({loader}: {loader: Ora}) => {
+    try {
+      const arch = process.arch === 'x64' ? 'amd64.' : '';
+      const installerUrl = `https://www.python.org/ftp/python/2.7.9/python-2.7.9.${arch}msi`;
```

Comment:
Can we detect if Chocolatey is available and provide user with a choice:
- install with Chocolatey
- install using official distribution (or whatever works here)

Asking, because it's a recommended way to install Node and Python2 according to the RN getting started guide

Reply:
I believe Chocolatey is the recommended because it will take care of setting environment variables, which I'm already doing. 

I debated using it underneath because it would have been easier, but there are some cons I couldn't ignore. Users need to:

* Have it installed (it's definitely not as popular as homebrew on mac, I don't have it for example)
* Run the script in powershell instead of regular prompt
* Run in an elevated prompt as a lot of packages will not get installed correctly unless they are run as an admin directly

What I'm doing is what chocolatey would do but without less limitations so not sure about the ROI.